CarpetLib/src/interpolate_3d_2tl.cc

```diff
@@ -104,11 +104,10 @@ void interpolate_3d_2tl(T const *restrict const src1, CCTK_REAL const t1,
   RT const s1fac = (t - t2) / (t1 - t2);
   RT const s2fac = (t - t1) / (t2 - t1);
 
-// Loop over region
-#pragma omp parallel
+  // Loop over region
+  // #pragma omp parallel
```

Comment:
this should not be committed in this form. Rather the whole pragma should be removed.